apps/dashboard/app/lib/account_cache.rb

```diff
@@ -9,10 +9,10 @@ module AccountCache
   # @return [Array<AccountInfo>] - the account info objects
   def accounts
     Rails.cache.fetch('account_info', expires_in: 4.hours) do
-      # only Slurm support in ood_core
+      # only Slurm & HTCondor support in ood_core
       # job_clusters: not to be confused with clusters
-      slurm_job_clusters = Configuration.job_clusters.select(&:slurm?)
-      slurm_job_clusters.empty? ? [] : slurm_job_clusters.map(&:job_adapter).flat_map(&:accounts).uniq
+      job_clusters = Configuration.job_clusters.find { |c| c.slurm? || c.htcondor? }
```

Comment:
`find` returns the first value that it can find. I believe `select` is the correct api to use here.